common/src/main/java/io/homeassistant/companion/android/common/assist/AssistViewModelBase.kt

```diff
@@ -114,21 +116,26 @@ abstract class AssistViewModelBase(
                     AssistPipelineEventType.STT_END -> {
                         stopRecording()
                         (it.data as? AssistPipelineSttEnd)?.sttOutput?.let { response ->
-                            onMessage(response["text"] as String, true, false)
+                            onMessage(response["text"] as String, true, false, false)
```

Comment:
I would like to push for a code style guideline in the codebase.
We should use named parameters when using booleans. Why is that an issue today
- The PR review is harder since you don't know at first sight what is each boolean in the function call. You need to find the function definition to know what it is.
- When using default params you might set the wrong boolean like `error` instead of `continueConversation` (I made the mistakes on this PR 🤦 and lost a lot of time debugging why my json is not properly parsed).

We could enforce that rule with ktlint.